src/ramses_tx/ramses.py

```diff
@@ -1339,6 +1339,19 @@
     },
 }
 
+# ventilation speed description
+_31D9_FAN_INFO: dict[int, str] = {
```

Comment:
is this for all 31d9, or only for some vendors?

Reply:
As far as I can tell, is only for Vasco/Climarad fans reporting speed via 31d9 as it was set by (faked) 4-button remotes. Note that "auto" mode is also in this dict, so can't use existing `_31DA_FAN_INFO` dict

Reply:
Renamed item to _31D9_FAN_INFO_VASCO to make this evident